test/functional/tools/composite_shapefile.xml

```diff
@@ -18,21 +18,21 @@
     <tests>
         <test>
             <param name="input" value="" ftype="shp" >
-                <composite_data value="shapefile/shapefile.dbf"/>
-                <composite_data value="shapefile/shapefile.prj"/>
                 <composite_data value="shapefile/shapefile.shp"/>
                 <composite_data value="shapefile/shapefile.shx"/>
+                <composite_data value="shapefile/shapefile.dbf"/>
+                <composite_data value="shapefile/shapefile.prj"/>
```

Comment:
And I was just looking at why the prj file looked like a binary file after the upload ...

Reply:
Do you know if this recently changed? It wired that you need to stick to the ordering of the datatype definition ... No tool dev should need to know this :(

Reply:
It's probably always been that way, otherwise there's a good chance existing tests would have broken.
I don't see anywhere in the code that we match up the filename to the composite file (but that's just glancing over the code)